tests/test_memory_service.py

```diff
@@ -0,0 +1,34 @@
+from uuid import uuid4
+
+
+def test_home(client):
+    resp = client.get("/")
+    assert resp.status_code == 200
+
+
+def test_create_memory(client):
+    data = {"content": "test memory"}
+    user = str(uuid4())
+
+    resp = client.post("/memories", json=data, headers={"user-id": user})
+
+    assert resp.status_code == 200
+    assert resp.json()["content"] == "test memory"
+
+
+def test_get_memory(client):
+    user = str(uuid4())
+
+    # Create memory
+    resp = client.post("/memories", json={"content": "hello"}, headers={"user-id": user})
+    memory_id = resp.json()["id"]
```

Comment:
I think you should first do some assertions, e.g. is the response a JSON actually, does it contain `id` attribute and only then extract to `memory_id`.

Reply:
This one too. I was meaning to check if response is a JSON object (probably there're some assertions in pytest for this)